gems/pending/fs/MiqFS/FsProbe.rb

```diff
@@ -14,6 +14,7 @@ module FsProbe
   PROBE_FILES.unshift("Ext4Probe")     if PROBE_FILES.delete("Ext4Probe")
   PROBE_FILES.unshift("Ext3Probe")     if PROBE_FILES.delete("Ext3Probe")
   PROBE_FILES.unshift("NTFSProbe")     if PROBE_FILES.delete("NTFSProbe")
+  PROBE_FILES.unshift("XFSProbe")      if PROBE_FILES.delete("XFSProbe")
```

Comment:
I don't understand. The version on master doesn't contain this line, so if it were inadvertently removed, it had to be merged.  If you're sure the line belongs there, then leave it, but it does worry me that it was deleted in the first place.